### ISSUE TITLE
Accuracy about your code with original paper

### ISSUE THREAD

**Issue by FangGet** [DecaYale/VLocNet#2]
Thanks for your opensource for VlocNet. Recently I'm also doing similar work. So there are some questions for the result:

1.  I wonder what accuracy (median translation and rotation error) do you obtain? Can you update ReadMe.md for that part?

2. you seems only trained on 7scenes, how about cambridge dataset?

3. as training needs groundtruth previous pose as input and testing needs predicted counterpart, I found it extremely easy to be overfit between gt and predicted, have you noticed this phenomenon?

**Comment by DecaYale** [comment]
Sorry about the confusion. This is an unofficial repo implemented by myself. There is no guarantee of the fidelity to the paper, although we have tried our best. The accuracy in our experiments is much lower than that reported in the paper. Moreover, the overfitting problem is also observed and haven't worked it out till now.

**Comment by FangGet** [comment]
I know its an unofficial implementation. I just wanna if you have done deeper work.
Any way, let's try our best to get it work.

Keep in touch.
